main.js

```diff
@@ -38,17 +38,26 @@ document.addEventListener("DOMContentLoaded", function () {
     }
   });
 
-  function createNewTodoItem(text) {
+  function createNewTodoItem(text, priority = "medium", isChecked = false) {
     const elem = document.createElement("li");
     elem.classList.add("flex-row");
 
     elem.innerHTML = `
         <label class="list-item">
-          <input type="checkbox" name="todoItem">
+          <input type="checkbox" name="todoItem" ${isChecked ? "checked" : ""}>
           <span class="checkmark"></span>
           <span class="text">${text}</span>
-          </label>
-          <button class="edit">Edit</button>
+          <select class="priority">
+       <option value="low" ${priority === "low" ? "selected" : ""}>Low</option>
+       <option value="medium" ${
+         priority === "medium" ? "selected" : ""
+       }>Medium</option>
+       <option value="high" ${
+         priority === "high" ? "selected" : ""
+       }>High</option>
+     </select>
```

Comment:
- To enhance readability of code, move this piece of code to a separate function or at least block.
- Create `priorities` array and use cycle for creating option-s

Reply:
Add different colors for each priority:
- Red for `high`
- Yellow for `medium`
- Blue for `low`

Reply:
I ccorrected these errors.